src/TriggerBinding/SqlTriggerScaleMonitor.cs

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Extensions.Sql.Telemetry;
+using static Microsoft.Azure.WebJobs.Extensions.Sql.Telemetry.Telemetry;
+using Microsoft.Azure.WebJobs.Host.Scale;
+using Microsoft.Extensions.Logging;
+using MoreLinq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Sql
+{
+    public class SqlTriggerScaleMonitor : IScaleMonitor<SqlTriggerMetrics>
```

Comment:
Add doc comment explaining what this class is used for (ideally with links to public docs about scaling)

Reply:
Done. Added general scaling doc for functions. Couldn't find anything on public docs specifically related to incremental scaling.